applications/crossbar/src/modules/cb_user_auth.erl

```diff
@@ -314,50 +341,73 @@ load_md5_results(Context, JObj) ->
     lager:debug("found MD5 credentials belong to user ~s", [wh_doc:id(JObj)]),
     cb_context:set_doc(Context, wh_json:get_value(<<"value">>, JObj)).
 
-%%--------------------------------------------------------------------
+
+%% @public
+-spec cleanup_reset_ids() -> 'ok'.
+cleanup_reset_ids() ->
+    CreatedBefore = wh_util:current_tstamp() - 2 * ?SECONDS_IN_DAY,
+    ViewOptions = [{'startkey', 0}
+                   ,{'endkey', CreatedBefore}
+                   %% TODO: find a limit that matches the number of pwd resets per day
+                   ,{'limit', couch_util:max_bulk_insert()}
+                   ,'include_docs'
+                  ],
+    case couch_mgr:get_results(?WH_ACCOUNTS_DB, ?LIST_BY_MTIME, ViewOptions) of
```

Comment:
I'm confused why the view is being looked up in `accounts`? AFAICT you are saving the reset docs back to the account database (and loading the user doc via `reset_id` pulls out the account id and uses that account database). Why would cleanup occur by querying the `accounts` database then? Those are account docs, not user docs.